crates/weaver_codegen_test/templates/registry/alt_weaver.yaml

```diff
@@ -2,4 +2,4 @@
 params:
   attributes: true
   metrics: false    # With this alternate configuration, the metrics are disabled
-  registry_prefix: "registry."
+  exclude_deprecated: true
```

Comment:
the default guideline is to generate deprecated attributes. I believe only golang doesn't generate them today.

So I'm not sure we need it. I have no objection having it, but if do, it should be `false` by default.

Reply:
Yes it's `false` by default.